driver.md

```diff
@@ -0,0 +1,70 @@
+Ethereum Simulation
+===================
+
```

Comment:
This file is a bit ad-hoc, and I tried to keep it minimal rather than general for now.